cypress/integration/with-security/helpers/generate_data.js

```diff
@@ -79,6 +79,10 @@ describe('Generating BWC test data with security', () => {
       .find('[data-test-subj="newItemButton"]')
       .click();
     cy.get('[data-test-subj="visType-timelion"]').click();
+    // update default expression to use `.es(*)` instead of `.opensearch(*)` for bwc
```

Comment:
NIT: do you think we could create a common function 

```
Cypress.Commands.add('clearWithBackspaces', (selector, count) => {
  cy.get(selector).then($element => {
    const backspaces = '{backspace}'.repeat(count);
    $element.type(`{selectAll}${backspaces}`);
  });
});
```

So this could simplifies to 
```
cy.clearWithBackspaces('[class="view-line"]', 15).type('.es(*)');
```
and this function could also be imported in other tests. I see there are 2 more tests using similar function. maybe put it in a until or common folder?

Also curious why we need these many backspaces

Reply:
Yeah, I meant to call this out in a comment. For some reason, the `{selectAll}` cypress sequence doesn't work in this field the way it's supposed to (or at least the way I understand that it's supposed to work). Previously, when the default expression was `.es(*)`, there were 6 total characters - you'll see that some tests "cleared" the input by repeating five backspaces (and reusing the `.`). Now that we've made the default expression `opensearch(*)`, there are 14 total characters that need deleting, if you delete them one by one. I didn't try `.clear()` either, but because it's supposed to just alias `type({selectAll}{backspace})`, I suspect it won't work either.

I do like the simplification idea, although I don't think we need a common function for it.

Reply:
See if you like my latest approach better! I sure do 😁

Reply:
kk I like it